kubernetes-samples/supporting-infrastructure/postinstall.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright Streamshub Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -eo pipefail
+
+STRIMZI_KAFKA=quay.io/strimzi/kafka:0.38.0-kafka-3.6.0
```

Comment:
Would be better to align it with Strimzi version used for managing Kafka cluster.

Reply:
It would, but `postinstall.sh` isn't actually used I copied it from Kroxylicious and should remove it from this PR.